hack/boilerplate/boilerplate.go.txt

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright (C) YEAR Red Hat, Inc.
+Copyright YEAR The Kubernetes Authors All rights reserved.
```

Comment:
-1 This is not going to work. Now the prerelease task is failing when it verified the headers. All files which don't get the reverted header still have "Copyright (C) 2016 Red Hat, Inc.". 

_boilerplate.py_ needs to change to handle both variations of the header. I would suggest to use a regular expressions in this case which allows the two variants "The Kubernetes Authors All rights reserved" and "Red Hat, Inc."